Util.hs

```diff
@@ -149,6 +172,10 @@ run name tool opts = do
             copyFile (name ++ "-fbuild.py") "fbuildroot.py"
             system_ $ "fbuild -j" ++ show p ++ " " ++ target ++ " > " ++ devNull
             removeFile "fbuildroot.py"
+        Gup -> do
+            createSymbolicLink (name ++ "-gup") "gup"
```

Comment:
"gup" is a directory here, so "copyFile" won't do. Copying the whole directory changes the modification dates on the gup scripts, which are automatically added as dependencies by Gup, thus triggering rebuilds for all targets; so that makes all the NoChange tests fail.

Note that the "pool" test uses GNU Parallel, for which I am not sure there is a windows version.